client/state/site-settings/selectors.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns true if we are requesting settings for the specified site ID, false otherwise.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {Boolean}        Whether site settings is being requested
+ */
+export function isRequestingSiteSettings( state, siteId ) {
+	return get( state.siteSettings.requesting, [ siteId ], false );
+}
+
+/**
+ * Returns true if we are saving settings for the specified site ID, false otherwise.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {Boolean}        Whether site settings is being requested
+ */
+export function isSavingSiteSettings( state, siteId ) {
```

Comment:
@aduth Same here, I did not create those reducers/selectors at first, but I needed them when trying to update the General Settings Form to use these actions/query https://t.co/h1heY6DV1e